config/src/main/java/org/axonframework/config/DefaultConfigurer.java

```diff
@@ -127,7 +129,8 @@ public class DefaultConfigurer implements Configurer {
 
     private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    private static final Runnable NOTHING = () -> {};
+    private static final Runnable NOTHING = () -> {
```

Comment:
I get why there are no configuration tests for the other test classes, but I would expect one for the `DefaultConfigurer` as it will carry the `SpanFactory` that is used throughout.